.gitattributes

```diff
@@ -1,2 +1,3 @@
-* text=auto
+* text=auto eol=lf
+*.* -eol
```

Comment:
This isn't needed and will cause files to be converted to platform-specific newlines.

Reply:
This is kinda needed? We might end up with a repository with mixed endings.

Reply:
The main rationale here was @TPGPL having issues with their PR, I triaged it to line endings for some reason. Keep in mind this is CRLF on a **Linux** Environment - this will cause issues.

Reply:
That's not what this line does.

Reply:
my bad for suggesting this change without an explanation---this PR title no longer has anything to do with the changes, we just wanted to fix https://github.com/ppy/osu-web/issues/3851. my suggested changes are meant to only force LF for files with no extension